### ISSUE TITLE
Doing a `quickWrite` to non-existing folder throws exception

### ISSUE THREAD

**Issue by Jandev** [CompositionalIT/farmer#248]
I was following the example and wanted to output the deployment file to a new folder called `deployment`, so came up with the following code
```fsharp
deployment 
|> Writer.quickWrite "..\\deployment\\development-environment"
```

The `deployment` folder does not exist, so the runnint the application throws an error
> Unhandled exception. System.IO.DirectoryNotFoundException: Could not find a part of the path 'D:\Projects\workstation\deployment\development-environment.json'.

While it's quite obvious and easy to create this folder, it would be nice if Farmer would create it by itself if a folder doesn't exist.
It's not a very important issue, but a nice-to-have and 'good first issue' for anyone who wants to contribute.

**Comment by isaacabraham** [comment]
Yep, this is a great first issue. Perhaps you would like to have a bash at it? :-) The function that needs changing is [here](https://github.com/CompositionalIT/farmer/blob/master/src/Farmer/Writer.fs#L31).

**Comment by Jandev** [comment]
I'm getting on it, right now!

**Comment by isaacabraham** [comment]
Fixed by #249
